{
  "uid" : "d1ade85bae2da61b",
  "name" : "\"before all\" hook: deploy implementation for \"cannot be initialized with a non-contract address\"",
  "fullName" : "Contract: TransparentUpgradeableProxy.\"before all\" hook: deploy implementation for \"cannot be initialized with a non-contract address\"",
  "historyId" : "38aab91f5b989c67a362277c0c9237d9",
  "time" : {
    "start" : 1635782704211,
    "stop" : 1635782704211,
    "duration" : 0
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04525b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04525b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/proxy/Proxy.behaviour.js:26:86)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: TransparentUpgradeableProxy"
  }, {
    "name" : "testClass",
    "value" : "Contract: TransparentUpgradeableProxy"
  }, {
    "name" : "testMethod",
    "value" : "\"before all\" hook: deploy implementation for \"cannot be initialized with a non-contract address\""
  }, {
    "name" : "package",
    "value" : "Contract: TransparentUpgradeableProxy"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "363a542d34e6efe6",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0683fd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782594932,
        "stop" : 1635782594932,
        "duration" : 0
      }
    }, {
      "uid" : "e39bb7dc285edfc6",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a044a130>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782593254,
        "stop" : 1635782593254,
        "duration" : 0
      }
    }, {
      "uid" : "ffb0d70908a708cb",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238bfd3670>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782592907,
        "stop" : 1635782592907,
        "duration" : 0
      }
    }, {
      "uid" : "53abe2dc1707ebed",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04482b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782565890,
        "stop" : 1635782565890,
        "duration" : 0
      }
    }, {
      "uid" : "71b141c6a80c4aa3",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a17de430>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782544178,
        "stop" : 1635782544178,
        "duration" : 0
      }
    }, {
      "uid" : "791b7a50c84e7b34",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238bfd3f70>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782514473,
        "stop" : 1635782514473,
        "duration" : 0
      }
    }, {
      "uid" : "f25a170599b5b34",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07c3790>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782502587,
        "stop" : 1635782502587,
        "duration" : 0
      }
    }, {
      "uid" : "79db9fc6e69157f4",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0fb4c10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782500192,
        "stop" : 1635782500192,
        "duration" : 0
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "aba1fa0a4e5eff31",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/8//#testresult/aba1fa0a4e5eff31",
        "status" : "broken",
        "statusDetails" : "Returned error: Transaction simulation failed: Error processing Instruction 0: invalid account data for instruction",
        "time" : {
          "start" : 1635589989606,
          "stop" : 1635589989606,
          "duration" : 0
        }
      }, {
        "uid" : "66b94903e0ffdb72",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/7//#testresult/66b94903e0ffdb72",
        "status" : "broken",
        "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a183cb20>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
        "time" : {
          "start" : 1635554979454,
          "stop" : 1635554979454,
          "duration" : 0
        }
      }, {
        "uid" : "aa7151bc017d2723",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/aa7151bc017d2723",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551253717,
          "stop" : 1635551253717,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "d1ade85bae2da61b.json",
  "parameterValues" : [ ]
}